transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
+"""
+
+from suds.sudsobject import Object as SudsObject
+
+
+class DnsEntry(SudsObject):
+    """
+    Representation of the DnsEntry object
+    """
+    TYPE_A = 'A'
+    TYPE_AAAA = 'AAAA'
+    TYPE_CNAME = 'CNAME'
+    TYPE_MX = 'MX'
+    TYPE_NS = 'NS'
+    TYPE_TXT = 'TXT'
+    TYPE_SRV = 'SRV'
```

Comment:
Actually, I simply copied these from the PHP API that can be downloaded from the TransIP website. Also, these are the only choices you have on their ControlPanel.